package_mod.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# /* this initializes the values for the different variables referenced in compatible_games */
+bash "../ModPackaging/utilities/ie_games.sh"
```

Comment:
I'm surprised this works as expected, as it starts a subshell. Typically one would use source or . here.

Reply:
Good point, I need to fix that one.  Will do so asap.